ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/WebSocketConfiguration.java

```diff
@@ -120,6 +121,14 @@ public void setAuthenticationPublicKeyFile(final File authenticationPublicKeyFil
     this.authenticationPublicKeyFile = authenticationPublicKeyFile;
   }
 
+  public String getAuthenticationAlgorithm() {
+    return authenticationAlgorithm;
+  }
+
+  public void setAuthenticationAlgorithm(final String algorithm) {
+    authenticationAlgorithm = algorithm;
```

Comment:
are we supporting choosing an algorithm for WebSockets auth? we would need another CLI option as well?

Reply:
I was under the impression that we use the same key for both websocket and http

Reply:
I missed a set of ATs entirely haha that's why this method is not used, will fix

Reply:
cbf81d7